wifiphisher/common/interfaces.py

```diff
@@ -465,6 +506,18 @@ def set_interface_mode(self, interface_name, mode):
         pyw.modeset(card, mode)
         pyw.up(card)
 
+    def add_card(self, interface):
```

Comment:
Instead of this it would be better to modify `is_interface_valid` to add a valid card to the active list. You will need to add a `internet` option to it.